### DISCUSSION TITLE
Atlas Scientific EZO ORP Sensor

### DISCUSSION THREAD

**Discussion by Rollipoli** [arendst/Tasmota#19250] [Support request] [answered]
Hi all.

I use 3 EZO Sensors, DO, PH and ORP, all 3 are shown in the web UI and reporting as they should.
After a restart only DO and PH starting properly, ORP is not comming back, blue and red flashing lights.

The console output after restart.

00:00:00.003 HDW: ESP32-D0WD-V3 
00:00:00.011 UFS: FlashFS mounted with 304 kB free
00:00:00.020 CFG: Loaded from File, Anzahl 475
00:00:00.025 QPC: Count 1
00:00:00.032 I2C: Bus1 using GPIO22(SCL) and GPIO21(SDA)
00:00:00.037 TFS: File 'mcp23x.dat' not found
00:00:00.092 BRY: Berry initialized, RAM used=4406 bytes
00:00:00.102 TFS: File '.drvset003' not found
00:00:00.102 CFG: Energy use defaults as file system not ready or file not found
00:00:00.104 SHT: ShutterMode: 1
00:00:00.116 SHT: ShutterMode: 1
00:00:00.117 SHT: ShutterMode: 1
00:00:00.118 SHT: ShutterMode: 1
00:00:00.130 Projekt tasmota - Tasmota Version 13.0.0(solo1)-2_0_10(2023.07.18 17:33:58)
00:00:00.200 I2C: BME280 found at 0x76
00:00:00.213 I2C: BME280 found at 0x77
00:00:00.215 I2C: BH1750 found at 0x23
00:00:00.218 I2C: INA219 found at 0x40
00:00:00.807 SCD40 serial nr 0x0 0x7F 0x0
00:00:00.808 I2C: SCD40 found at 0x62
00:00:01.552 WIF: verbinden mit AP1 gh-reisinger-intern Channel 11 BSSId AC:84:C6:F1:51:B6 in Modus 11n als tasmota-0FD67C-5756...
00:00:02.011 I2C: EZODO found at 0x61
00:00:02.018 I2C: EZOpH found at 0x63

The output of i2cscan after restart is showing the 3 Sensors

08:11:25.738 CMD: i2cscan
08:11:25.768 MQT: stat/tasmota_0FD67C/RESULT = {"I2CScan":"Device(s) found on bus1 at 0x23 0x40 0x61 0x62 0x63 0x76 0x77"}

The ORP sensor is comming back only after power cycle 

The console output after powercycle.

00:00:00.003 HDW: ESP32-D0WD-V3 
00:00:00.009 UFS: FlashFS mounted with 304 kB free
00:00:00.015 CFG: Loaded from File, Anzahl 474
00:00:00.021 QPC: Count 1
00:00:00.029 I2C: Bus1 using GPIO22(SCL) and GPIO21(SDA)
00:00:00.032 TFS: File 'mcp23x.dat' not found
00:00:00.081 BRY: Berry initialized, RAM used=4406 bytes
00:00:00.090 TFS: File '.drvset003' not found
00:00:00.090 CFG: Energy use defaults as file system not ready or file not found
00:00:00.092 SHT: ShutterMode: 1
00:00:00.104 SHT: ShutterMode: 1
00:00:00.105 SHT: ShutterMode: 1
00:00:00.106 SHT: ShutterMode: 1
00:00:00.118 Projekt tasmota - Tasmota Version 13.0.0(solo1)-2_0_10(2023.07.18 17:33:58)
00:00:00.188 I2C: BME280 found at 0x76
00:00:00.201 I2C: BME280 found at 0x77
00:00:00.203 I2C: BH1750 found at 0x23
00:00:00.206 I2C: INA219 found at 0x40
00:00:00.951 WIF: verbinden mit AP1 gh-reisinger-intern Channel 11 BSSId AC:84:C6:F1:51:B6 in Modus 11n als tasmota-0FD67C-5756...
00:00:02.018 I2C: EZODO found at 0x61
00:00:02.025 I2C: EZOORP found at 0x62
00:00:02.032 I2C: EZOpH found at 0x63

**Comment by sfromis** [comment, marked as answer]
`I2C: SCD40 found at 0x62` 

Do you have a SCD40 (or similar) sensor connected? If yes, it does not work with multiple devices and conflicting addresses. If no, disable the SCD40 driver using the command `I2cDriver62 0`

**Reply by Rollipoli** [reply]
No, there is no SCD40 Sensor connected, disable the SCD40 driver worked!!!
Thank you very much!
